src/request_handlers/submit.rs

```diff
@@ -30,6 +30,8 @@ pub struct SubmitChat {
 
 impl SubmitChat {
     pub async fn get_app_info(&self, conn: &mut SqliteConnection) -> sqlx::Result<AppInfo> {
+        // Didn't check if this replacement is correct
+        // (underlying function changed)
```

Comment:
@hpk42 and me agreed, that we will keep the whole submission system out of prs for now, so I will for now not investigate this further